src/components/login/index.tsx

```diff
@@ -1,19 +1,48 @@
 import React, { useState } from 'react';
 import { Input } from '../input';
 import { Button } from '../button';
+import { isValidEmail, isValidPassword } from '../../utils/strings-utils';
 
 export const Login = () => {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
+  const [emailError, setEmailError] = useState('');
+  const [passwordError, setPasswordError] = useState('');
+
+  const validateFields = () => {
+    setEmailError('');
+    setPasswordError('');
+
+    if (!email.trim()) {
```

Comment:
embora isso funcione, talvez não seja a melhor opçaõ em termos de legibilidade. 
acho que testar uma conversão de email pra boolean talvez fosse melhor e ficaria mais simples:

```suggestion
    if (!email) {
```

Reply:
ou, talvez:

```suggestion
    if (!email.lenght) {
```

Reply:
certo, usei a sua sugestão  if (!email.lenght).